app/assets/stylesheets/static_pages.css.scss

```diff
@@ -0,0 +1,59 @@
+*{
+  padding:0;
+  margin:0;
+}
+
+#container {
+  margin: 48px auto;
+  width: 960px;
+
+  section {
+    clear: both;
+    p {
+      font-size: 22px;
+    }
+  }
+
+  header {
+    margin-bottom: 46px;
+
+    h1 {
+      float: left;
+      margin: 0;
+    }
+  }
+}
+
+.navigation-links {
+  float: right;
+
+  li {
+    display: inline-block;
+    margin: 0 16px;
+  }
+}
+
+.navigation-links li {
+  display: inline-block;
+  margin: 0 16px;
+}
+
+/* MISC */
+
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+
+html[xmlns].clearfix{
```

Comment:
Avoid qualifying class selectors with an element.<br>Avoid qualifying attribute selectors with an element.<br>Opening curly brace `{` should be preceded by one space